### PR TITLE
Webhook for subs rolling

### PR DESCRIPTION
**Pull request by gausie**
- Remove /subs command
- Add webhook for subs rolling. Thanks @cdmoyer

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -26,6 +27,25 @@ app
...
+    const token = req.query.token;
+    const SUBS_ROLLING_TOKEN = process.env.SUBS_ROLLING_TOKEN!;
+
+    if (!token) return res.status(StatusCodes.UNAUTHORIZED).json({ error: "No token" });
+    if (token !== SUBS_ROLLING_TOKEN)
+      return res.status(StatusCodes.FORBIDDEN).json({ error: "Invalid token" });
+
+    try {
+      await rollSubs();
+      return res.status(StatusCodes.OK).json({ status: "Thanks Chris!" });
```
Comment (docrostov) [inline src/server.ts:41]: thanks chris

File: src/subs.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    );
+    throw new Error("Something is configured wrong");
+  }
+
+  const subRollEmoji = guild.emojis.cache.find((e) => e.name === "subsRolling") || "";
+
+  await iotmChannel.send({
+    content: `🚨${subRollEmoji} Attention ${roleMention(
+      SUBSCRIBER_ROLE_ID,
+    )}! This is an automated message to let you know that subscriptions are now rolling ${subRollEmoji}🚨`,
```
Comment (docrostov) [inline src/subs.ts:25]: i wish we'd found a better subsrolling emoji but this is fine, users will spam sandwich photos as soon as it drops

**Review by docrostov — APPROVED** [review]
thank you blessed Chris Dhris Mhis (Coyer Doyer Moyer)

File: src/subs.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { roleMention } from "discord.js";
+
+import { discordClient } from "./clients/discord.js";
+
+export async function rollSubs() {
+  const GUILD_ID = process.env.GUILD_ID!;
```
Comment (horrible-little-slime) [inline src/subs.ts:6]: Aren't these traditionally top level for most discord client things?

Why aren't they here?

File: src/subs.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  const iotmChannel = guild?.channels.cache.get(IOTM_CHANNEL_ID);
+
+  if (!iotmChannel?.isTextBased()) {
+    await discordClient.alert(
+      "Someone has tried to hit a subs rolling webhook but the guild or iotm channel are incorrectly configured",
+    );
+    throw new Error("Something is configured wrong");
+  }
+
+  const subRollEmoji = guild.emojis.cache.find((e) => e.name === "subsRolling") || "";
```
Comment (horrible-little-slime) [inline src/subs.ts:20]:
```suggestion
  const subRollEmoji = guild.emojis.cache.find((e) => e.name === "subsRolling") ?? "";
```

File: src/subs.ts (hunk 2 above)
Comment (horrible-little-slime) [reply src/subs.ts:25]: as long as we have an emoji named `subsRolling` it'll work, so we can switch to a new one whenever

File: src/subs.ts (hunk 4 above)
Comment (gausie) [reply src/subs.ts:20]: Not necessary at all but sure

File: src/subs.ts (hunk 3 above)
Comment (gausie) [reply src/subs.ts:6]: Because of a quick refactor. Will fix later as I'm planning on revamping config variables soon

**Pull request merged**